routes/home.py

```diff
@@ -0,0 +1,36 @@
+# This is backwards. The controller should request what routes it needs. 
```

Comment:
I didn't spend much time on this part(Actually I didn't spent much time in anything) and this can have some flaw. Having that said, your whole implementation is wrong 😄 

1 - HomeRouter? You should have an application router and many routes
2- On my version there was no coupling between router and controllers, but there was string route coupled to a controller, that is all
3 -  The mapping route to a controller could be done in a factory, again, I didn't spend much time on this and you did figure it out either